addons/mail/populate/mail_message.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo.tools import populate
+
+
+class Message(models.Model):
+    _inherit = "mail.message"
+    _populate_dependencies = ["res.partner"]
+
+    _populate_sizes = {
+        'small': 1e4,
+        'medium': 1e7,
+        'large': 1.5e8,
+    }
+
+    def _populate_factories(self):
+        partner_ids = self.env.registry.populated_models['res.partner']
+
+        def get_res_id(values, counter, random):
+            if values['model'] == 'res.partner':
+                return random.choice(partner_ids)
+            return False
+
+        return [
+            ('author_id', populate.randomize(partner_ids)),
+            ('body', populate.constant('<p>Message {counter}</p>')),
```

Comment:
Just asking because I don't know, but doesn't populate have tools to create made up words and sentences instead? That would feel more natural than this very programmatic approach.

Reply:
Hello @seb-odoo 
We don't have such a function in populate to generate names, that's why I have generated this kind of names.
Thanks